src/machines/java/com/enderio/machines/common/lang/MachineLang.java

```diff
@@ -18,5 +18,11 @@ public class MachineLang {
 
     public static final Component TOOLTIP_ENERGY_EQUIVALENCE = REGISTRATE.addLang("gui", EnderIO.loc("energy_equivalence"), "A unit of energy, equivalent to FE.");
 
+    public static final Component TOO_MANY_MOB = REGISTRATE.addLang("gui", EnderIO.loc("spawner.too_many_mob"), "Too many Mobs");
```

Comment:
Capitalisation of Mobs and Spawners is likely unnecessary here; but could be stylistic preference on my part.